lib/dom-parser.js

```diff
@@ -49,98 +43,82 @@ function normalizeLineEndings(input) {
  *
- * _xmldom is different from the spec in that it allows an `options` parameter,
- * to control the behavior._
+ * _xmldom is different from the spec in that it allows an `options` parameter, to control the behavior._.
  *
+ * @class
  * @param {DOMParserOptions} [options]
- * @constructor
- *
  * @see https://developer.mozilla.org/en-US/docs/Web/API/DOMParser
  * @see https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#dom-parsing-and-serialization
  */
 function DOMParser(options) {
 	options = options || { locator: true };
 
 	/**
-	 * The method to use instead of `Object.assign` (defaults to or `conventions.assign`),
-	 * which is used to copy values from the options before they are used for parsing.
+	 * The method to use instead of `Object.assign` (defaults to or `conventions.assign`), which is used to copy values from the
+	 * options before they are used for parsing.
 	 *
-	 * @type {function (target: object, source: object | null | undefined): object}
+	 * @type {function (target: Object, source: Object | null | undefined): Object}
+	 * @access private
+	 * @see Conventions.assign.
 	 * @readonly
-	 * @private
-	 * @see conventions.assign
 	 */
 	this.assign = options.assign || conventions.assign;
 
 	/**
-	 * For internal testing: The class for creating an instance for handling events from the SAX parser.
-	 * __**Warning: By configuring a faulty implementation, the specified behavior can completely be broken.**__
+	 * For internal testing: The class for creating an instance for handling events from the SAX parser. ****Warning: By configuring
+	 * a faulty implementation, the specified behavior can completely be broken.****
 	 *
 	 * @type {typeof DOMHandler}
+	 * @access private
 	 * @readonly
-	 * @private
 	 */
 	this.domHandler = options.domHandler || DOMHandler;
 
 	/**
 	 * A function that is invoked for every error that occurs during parsing.
 	 *
-	 * If it is not provided, all errors are reported to `console.error`
-	 * and only `fatalError`s are thrown as a `ParseError`,
-	 * which prevents any further processing.
-	 * If the provided method throws, a `ParserError` is thrown,
-	 * which prevents any further processing.
+	 * If it is not provided, all errors are reported to `console.error` and only `fatalError`s are thrown as a `ParseError`, which
+	 * prevents any further processing. If the provided method throws, a `ParserError` is thrown, which prevents any further
+	 * processing.
 	 *
-	 * Be aware that many `warning`s are considered an error
-	 * that prevents further processing in most implementations.
+	 * Be aware that many `warning`s are considered an error that prevents further processing in most implementations.
 	 *
 	 * @type {function(level:ErrorLevel, message:string, context: DOMHandler):void}
-	 *
-	 * @see onErrorStopParsing
-	 * @see onWarningStopParsing
+	 * @see OnErrorStopParsing.
+	 * @see OnWarningStopParsing.
```

Comment:
@shunkica @homer0 Is camel casing the first letter coming from the plugin?
This would be a no-go for me. Or do we need to keep those references to imported things in single quotes to indicate that the plugin shouldn't tough those?

Reply:
This is the curly brackets thing I mentioned in the other comment. It thinks that those types are description, and since `jsdocEnsureDescriptionsAreSentences` is enabled by default, it's making the first letter uppercase, and ensuring there's a period at the end.

Reply:
It is, we need to add `@link` for what you want to achieve.